.circleci/config.yml

```diff
@@ -714,12 +714,16 @@ jobs:
         environment:
           # we need CGO because of go-sqlite3
           CGO_ENABLED: 1
+          # Reduce golangci-lint memory usage (default is 100)
+          GOGC: 20
     working_directory: /go/src/github.com/grafana/grafana
     steps:
       - checkout
       - run:
           name: Lint Go
-          command: "make lint-go"
+          command: |
+            make scripts/go/bin/golangci-lint
```

Comment:
I think this looks weird. But I'm not a make expert :)

Reply:
@bergquist It's not weird, it tells `make` to build scripts/go/bin/golangci-lint, which is just the executable :)